src/rules/ban_untagged_ignore.rs

```diff
@@ -37,13 +37,32 @@ impl LintRule for BanUntaggedIgnore {
       .collect();
 
     for span in violated_spans {
-      context.add_diagnostic(
+      context.add_diagnostic_with_hint(
         span,
         "ban-untagged-ignore",
-        "Ignore directive requires lint rule code",
+        "Ignore directive requires lint rule name(s)",
+        "Add one or more lint rule names.  E.g. // deno-lint-ignore adjacent-overload-signatures",
       )
     }
   }
+
+  fn docs(&self) -> &'static str {
+    r#"Requires `deno-lint-ignore` to be annotated with one or more rule names.
+
+Ignoring all rules can mask unexpected or future problems.  Therefore you need to explicitly specify which rule(s) are to be ignored.
```

Comment:
```suggestion
Ignoring all rules can mask unexpected or future problems. Therefore you need to explicitly specify which rule(s) are to be ignored.
```

redundant space :eyes: